### DISCUSSION TITLE
Desafios com CSS

### DISCUSSION THREAD

**Discussion by afonsopacifer** [frontendbr/forum#238] [Discussão]
Hey, 
Hoje, após uma sugestão da @nathpaiva, iniciei uma série de propostas de desafios com CSS, bem ao estilo dos que o Maoujor e o Bernard faziam ❤️ 

Bem, vou postar os desafios no meu site, e no meu canal no youtube, mas pensei em abrir essa issue para postar aqui tbm, já que não dá para centralizar nada no Facebook direito e talz..

Enfim.. o primeiro DESAFIO é criar a logo do Instagram apenas com CSS 💙 

![instagram-logo](https://cloud.githubusercontent.com/assets/6748866/16891310/47c57612-4ace-11e6-9181-9a10fbdea2f9.jpg)
- **Nível:** Médio
- **Tempo em que foi criado:** 15 minutos
- **Tempo estimado proposto:** 30 minutos
- **Deadline do desafio:** 22/07/16

Artigo com as regras:
http://afonsopacifer.com/desafio-01-instagram-logo.html

Video falando sobre o desafio:
https://www.youtube.com/watch?v=7qGvEwSpxyo

Quem topa?

**Comment by AdrianoCahete** [comment]
Precisa gravar fazendo ou só fazer tá de boa?

**Comment by afonsopacifer** [comment]
Só fazer e enviar o link, de boas, o importante é a diversão e aprendizado :)

**Comment by AdrianoCahete** [comment]
Fazendo aqui, conheci/estudei algumas coisas que nunca tinha feito.... Seria legal também pedir pro pessoal dizer o que aprendeu/melhorou tecnicamente.

**Comment by afonsopacifer** [comment]
@AdrianoCahete, o objetivo é esse mesmo, fazer o pessoal mexer e aprender com coisas novas :)
O lance do tempo, é meio que uma questão individual, cada um vê o tempo proposto e compara com o tempo que levou, assim, vc tem um noção melhor de como está, mas o que realmente importa é ir treinando, aprendendo e se mantendo ativo com diversão <3

No fim das contas, é um desafio contara você mesmo :)

**Comment by napalmdev** [comment]
Cara, visto a quantidade de projetos do tipo no codepen e tal, poderia fazer assim: 
Vc propõe o desafio, a galera faz e entrega até a deadline, aí vc analisa e posta as melhores soluções pra galera poder analisar soluções superiores e entenderem o que poderiam ter feito de diferente, assim como nos Katas do Codewars. Seria muito foda.

Seria o CSSWars   :D

**Comment by afonsopacifer** [comment]
@napalmdev haha Boa, eu pretendo adicionar no post do desafio todos os links do pessoal. 

Agora que vc falou, acho que poderia sim dar um destaque para os que fizeram das formas mais criativas, assim o pessoal poderia ver com mais facilidade as formas diferentes de se fazer a mesma coisa. Boa dica mano :)

**Comment by napalmdev** [comment]
Exato @afonsopacifer, por exemplo no codewars eu sempre terminava os Katas e verificava as soluções mais bem avaliadas, e acabava tendo altos insights, tipo, carambaaa como nao pensei nisso, principalmente soluções com map, reduce e filter, e eu utilizando for kkkkkkkkkkkkkkkk

**Comment by afonsopacifer** [comment]
hahahahahaa, realmente, agora que vc falou, acho que vou adicionar uma lance de votos ou like seila, para a galera ver e marcar os que gostaram... só implementar "rapidinho" afinal, dormir para q? rsrsrsr

Mas talvez seja uma boa algo assim, vamos ver meu sono hoje rsrsr

**Comment by AdrianoCahete** [comment]
> acho que vou adicionar uma lance de votos ou like 

Se for fazer algo do tipo, deixa que o projeto/código seja anônimo, pra não acabarem votando em alguém mais famoso ou só em quem gostem e passem a votar em cima do código. Por incrível que pareça, acontece bastante...

**Comment by napalmdev** [comment]
hsadudsuasduhaha dormir pra quê neh @afonsopacifer , e @AdrianoCahete é um caso a se pensar mesmo

**Comment by olegon** [comment]
Médio?! Para mim isso é difícil! Mas vamos lá!  \o/

> Tudo parece impossível até que seja feito. - Mandela

**Comment by leandroruel** [comment]
adiciona na dificuldade fazer a logo com 1 so div

**Comment by adeonir** [comment]
poutz... trabalhoso conseguir acertar esse gradient

**Comment by doug2k1** [comment]
Muito legal! Aprendi muito sobre gradient fazendo este desafio!

**Comment by adeonir** [comment]
@afonsopacifer, dá um conferida no teu email... mandei esses dias e ainda não tive resposta de recebido

**Comment by afonsopacifer** [comment]
Opa, @adeonir recebi sim, é que ainda não tive tempo de responder todos os e-mail novos que chegaram :)

ps: O seu ficou irado 👍

**Comment by DevDalee** [comment]
Conseguiiii!
Não foi em 30 mins, mas para quem tá a mais de 6 meses parado foi muito bom!!
